packages/codegen.go/src/models.ts

```diff
@@ -618,13 +618,13 @@ class ModelDef {
   readonly Name: string;
   readonly Format: go.ModelFormat;
   readonly Description?: string;
-  readonly Fields?: Array<go.ModelField>;
+  readonly Fields: Array<go.ModelField>;
```

Comment:
This was a missed case when we refactored the Go code model to remove optional arrays.